server/src/updaters/thread-permission-updaters.js

```diff
@@ -69,6 +69,7 @@ async function changeRole(
     dbQuery(membershipQuery),
     changeRoleThreadQuery(threadID, role),
   ]);
+
```

Comment:
Nit: let's avoid making purely whitespace changes to files in a PR. The issue is that it pollutes version control history. We want it to be easy for somebody to [look at a file's history](https://github.com/Ashoat/squadcal/commits/master/server/src/updaters/thread-permission-updaters.js) and see when it's changed.